client/my-sites/stats/stats-module/index.jsx

```diff
@@ -173,12 +173,13 @@ class StatsModule extends Component {
 				) }
 				{ shouldShowNewModule && (
 					<>
+						{ /* TODO: Move this header to the summary page when modernising it */ }
```

Comment:
Moving this header to the summary page will be beneficial because it will unify all of the summary pages. The only thing left is to handle `path` for all cases, so it could be a part of modernising the Summary page.